src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -1492,6 +2282,195 @@ static void Main(string[] args)
             VerifyOperationTreeAndDiagnosticsForTest<VariableDeclarationSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [Fact]
+        public void UsingBlock_InvalidIgnoredDimensions_SwitchExpression()
+        {
+            string source = @"
+class C
+{
+    void M1()
+    {
+        int y = 10;
+       using( /*<bind>*/int[] x = new int[0]/*</bind>*/){}
+    }
+}
+";
+            var syntaxTree = Parse(source, filename: "file.cs");
+            var rankSpecifierOld = syntaxTree.GetCompilationUnitRoot().DescendantNodes().OfType<ArrayRankSpecifierSyntax>().First();
+            var rankSpecifierNew = rankSpecifierOld
+                .WithSizes(SyntaxFactory.SeparatedList<ExpressionSyntax>(SyntaxFactory.NodeOrTokenList(SyntaxFactory.ParseExpression("y switch { int z => 42 }"))));
+            syntaxTree = syntaxTree.GetCompilationUnitRoot().ReplaceNode(rankSpecifierOld, rankSpecifierNew).SyntaxTree;
+
+
+            string expectedOperationTree = @"
+	IVariableDeclarationOperation (1 declarators) (OperationKind.VariableDeclaration, Type: null, IsInvalid) (Syntax: 'int[y switc ...  new int[0]')
```

Comment:
>	 [](start = 0, length = 1)

Can you please turn all the tabs into spaces?  Alternatively, remove the tabs.  Indentation here currently has a mixture of tabs and spaces.